about.html

```diff
@@ -0,0 +1,117 @@
+                            <a href="#sponsor">Sponsors</a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="menu-desktop">
+                    <div class="menu-atlas">
+                        <img src="./img/logo.png" alt="Menu Icon">
+
+                    </div>
+                    <ul class="menu-ul">
+                        <li class="menu-li">
+                            <a href="./index.html">Home</a>
+                        </li>
+                        <li class="menu-li">Tickets</li>
+                        <li class="menu-li">Join</li>
+                        <li class="menu-li">Sponsor</li>
+                        <li class="menu-li">News</li>
+                        <li class="menu-li cc">CC Campaign</li>
+
+                    </ul>
+                </div>
+
+            </nav>
+        </div>
+
+    </header>
+    <main>
+        <section>
+            <div class="about-section">
+                <div class="intro">
+                    <h1 class="about-h1"> "CREATIVE COMMONS <br>GLOBAL SUMMIT ,<br> 2015"</h1>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eveniet voluptatum vero velit, ad
+                        incidunt ex, eligendi aliquid suscipit doloremque culpa nihil tempore placeat nostrum veniam
+                        similique aliquam rerum nisi neque atque. Unde quas iure et id illo necessitatibus debitis fuga
+                        expedita illum impedit voluptate veniam in molestias itaque tenetur quia ipsam, dignissimos
+                        ipsum, esse ratione eveniet obcaecati. A nemo ducimus mollitia tenetur officiis eveniet, dolores
+                        unde asperiores iste optio accusamus possimus laudantium qui! Nihil saepe totam numquam magnam
+                        aperiam quasi.</p>
+
+                    <h6>Please Contack us if you need any furthur information about this Bootcamp</h6>
+
+
+                </div>
+            </div>
+            <div class="about-our-logo">
+                <h2> CREATIVE COMMONS GLOBAL SUMMIT 2015 LOGO</h2>
+                <p> Lorem ipsum dolor sit amet consectetur adipisicing elit. Possimus animi veritatis rem nesciunt
+                    temporibus, adipisci expedita voluptatibus culpa illo saepe et nisi suscipit quidem voluptatem!
+                    Tempora aliquam rem necessitatibus facilis.</p>
+                <div class="global-logo">
+                    <img src="./img/logo.png" alt="Logo" width="200px">
+                </div>
+
+
+
+        </section>
+        <section class="logo-section">
+            <div class="see-past">
+                <h3>SEE THE PAST CC GLOBAL SUMMITS</h3>
+                <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Perspiciatis ut accusamus dignissimos sed
+                    deleniti natus laborum id voluptatibus atque voluptatum? Ipsa, ab. Placeat molestias pariatur
```

Comment:
[Optional] I suggest that you adjust the width of the paragraph text to match the design. I'm sure you'll appreciate the look.

Your design:
![image](https://user-images.githubusercontent.com/20554583/203925509-16f27cfe-9d23-493c-969a-e3acff23fe69.png)


Cindy's design:
![image](https://user-images.githubusercontent.com/20554583/203925551-205847a9-c065-431e-8942-5d6e4d90427a.png)